src/openrct2/interface/Window_internal.h

```diff
@@ -176,6 +176,8 @@ struct WindowBase
 
     void ResizeFrame();
     void ResizeFrameWithPage();
+
+    void ResizeSpinner(WidgetIndex widgetIndex, const ScreenCoordsXY& origin, int32_t widget_width);
```

Comment:
```suggestion
    void ResizeSpinner(WidgetIndex widgetIndex, const ScreenCoordsXY& origin, int32_t width);
```
Ideally this also becomes 
```suggestion
    void ResizeSpinner(WidgetIndex widgetIndex, const ScreenCoordsXY& origin, int32_t width, int32_t height);
```
We should start de-hardcoding all those sizes, it can keep the constants for default size but the resize functionality shouldn't be aware of such things.

Reply:
That was the original design, but it changed due to this comment from @Gymnasiast 

https://github.com/OpenRCT2/OpenRCT2/pull/20479#discussion_r1242775179

Reply:
I see, still please change it to have width and height as the parameters, a resize function shouldn't be one dimensional.

Reply:
I've rebased and made this change.